### ISSUE TITLE
Ignoring of S3 Glacier type errors

### ISSUE THREAD

**Issue by Flood-ctrl** [Azure/azure-storage-azcopy#2834]
Hello is it possible to add some options or parameter for ignoring S3 Glacier type, I had a bunch of objects in the S3 and I was needed to move the data to AZ Storage account by using azcopy copy and during the copy I had a lot of failed (and canceled by azcopy) attempts due to objects were already moved to S3 Glacier and azcopy shows the error: **RESPONSE 403: 403 Forbidden** ; **ERROR CODE: CannotVerifyCopySource** ; and operation was canceled. It will be great if azcopy will be able to skip or/and ignore the errors related to Glacier type. 
Thank you,
Serhii

**Comment by momodedev** [comment]
### INFO: Skipping S3 object launch.json, as it is not a valid AWS storage class: DEEP_ARCHIVE

### failed to perform copy command due to error: cannot start job due to error: Invalid AWS Storage Class: DEEP_ARCHIVE.

Is the above output message your expectation?

**Comment by Flood-ctrl** [comment]
If it doesn’t fail the entire copy process and only notifies about the unavailability of the copying object due to the S3 storage class, then that’s fine. Thank you.

**Comment by momodedev** [comment]
Yes, it is. the change just output the message as mentioned above.

**Comment by Flood-ctrl** [comment]
Sounds good, cause my concerns were mostly about the interruption of copy process if issues with S3 storage class appear if the fix will ignore this issue and will proceed with the copy then great. Thank you

**Comment by momodedev** [comment]
I can send the azcopy.exe to you. You might test it in your environment.

**Comment by Flood-ctrl** [comment]
Unfortunately I'm not able to test it cause I don't have sanitized data
